lib/puppet/provider/image_streamer_golden_image/image_streamer.rb

```diff
@@ -21,32 +21,29 @@
 
   mk_resource_methods
 
-  def exists?
-    super([nil, :found, :download_details_archive, :download])
```

Comment:
Why that was removed from this module?
Similar `exists?` weren't changed, for example `oneview_volume/c7000.rb`